CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 #### 0.5.1 (Next)
 
+* [#45](https://github.com/slack-ruby/slack-ruby-bot-server/pull/45): Update dependencies so we can follow the latest `representable`. - [@swalberg](https://github.com/swalberg)
```

Comment:
There's an extra `.`, it belongs at the end ;)

The log is not very clear, I would just make this _Updated grape-roar to 0.4.0_